app/views/distributions/index.html.erb

```diff
@@ -99,7 +99,8 @@
               <tr>
                 <th>ID</th>
                 <th>Partner</th>
-                <th class="date">Date of Distribution</th>
+                <th class="date">Initial Allocation</th>
+                <th class="date">Distribution Scheduled for</th>
```

Comment:
Do you need the "Distribution" part of this header? It makes the page wider and doesn't add any more information (we're already on the Distributions page).

Reply:
Think that's a product question from @cielf

Reply:
@dorner has a good point in this case.  Please do remove the "Distribution" from "Distribution Scheduled for".